fast_api_als/main.py

```diff
@@ -1,9 +1,13 @@
 import time
+import logging
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 from fast_api_als.routers import users, submit_lead, lead_conversion, reinforcement, oem, three_pl, quicksight
 
+
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
```

Comment:
you can create a separate util for your custom logger and import that module wherever required